public/electron.js

```diff
@@ -17,6 +17,18 @@ Sentry.init({
   release: process.env.npm_package_version
 })
 
+let tray = null
+let iconOS = null;
+let trayIcon = null;
+let msgCheck = false;
```

Comment:
We should save this variable in local storage. So, we would load it after user restarts the wallet.